src/main/java/com/example/icecreamup/IcecreamupController.java

```diff
@@ -56,6 +54,7 @@ public String orderForm(Model model) {
 	public String saveOrder(OrderDTO orderDTO) {
 		orderDTO.setTimeStamp(new Date());
 		
+		//Provides error handling for the save method
```

Comment:
It was a good idea to add a try-catch statement here so that an error will be returned if the save function fails to write to the database.  I added additional Javadoc to all files.